src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerSøk.tsx

```diff
@@ -137,10 +137,13 @@ const FagsakDeltagerSøk: React.FC = () => {
                     stil={'lys'}
                 />
             )}
-            <OpprettFagsakModal
-                søkeresultat={deltagerForOpprettFagsak}
-                lukkModal={() => settDeltagerForOpprettFagsak(undefined)}
-            />
+            {deltagerForOpprettFagsak && (
```

Comment:
Synes det var litt forvirrende med både denne og visModal. Men ser ut som når deltagerForOpprettFagsak finnes så skal modalen alltid være åpen? Kunne man forenklet litt og droppet både visModal og lukkModal?

F.eks 
- Sette open til true i selve modalen
- Sende inn settDeltagerForOpptrettFagsak som prop i `OpprettFagsakModal` og kalle den i onClose? i selve modalen?

Reply:
Enig i at denne er forvirrende 😅 jeg kan droppe å sende med visModal, men forslag nummer to blir litt rart det andre stedet som OpprettFagsakModal blir kalt, hvor man faktisk setter visModal til false:
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/26d4589b-f21e-44a6-b013-d9874362eb14)

Foreslår å fikse visModal-delen nå, så kan jeg se på strukturen på denne modalen i en annen PR siden det ikke har noe mer oppgraderingene her å gjøre!